app/src/main/java/mozilla/lockbox/presenter/ItemListPresenter.kt

```diff
@@ -37,7 +37,9 @@ interface ItemListView {
     val sortItemSelection: Observable<Setting.ItemListSort>
     fun updateItems(itemList: List<ItemViewModel>)
     fun updateItemListSort(sort: Setting.ItemListSort)
-    fun setDisplayName(text: String)
+    fun setDisplayEmailName(text: String)
+    fun setAccountName(text: String)
+    fun setAvatarFromURL(url: String)
```

Comment:
I think I'd prefer `var` properties here.  Setters like this are more reminiscent of idiomatic Java than Kotlin.

```java
public void setProperty(Property property) {}
```

:-P